test/e2e/scheduling/preemption.go

```diff
@@ -24,7 +24,7 @@ import (
 	"k8s.io/client-go/tools/cache"
 
 	appsv1 "k8s.io/api/apps/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
`v1` is unnecessary alias.

Reply:
goimports adds this now when there are other imports that could conflict if they didn't have aliases
https://github.com/golang/go/issues/30051

Reply:
I read it differently - my understanding is some package like `k8s.io/v1` is not allowed, but `k8s.io/api/core/v1` should be good.

With regards to the unnecessary alias, are you using vscode and hence it auto generate the v1 alias?

Reply:
yes, I do use vscode run goimports on save

Reply:
If goimports does it, I don't see a reason to avoid it.  We generally defer to the golang ecosystem choices on formatting style except where they conflict with previous decisions.  I don't think we've ever specifically prevented basic aliases.

The standard convention is for this to be `corev1` though, not `v1`.  So probably better to flip the package to use that.

Reply:
try to change to "gofmt" on settings "Go: Format Tool".

Reply:
@smarterclayton I think we're good to have "v1" as the _package path_ - just like k8s did before. The restriction only applies onto _module path_. See comment [here](https://github.com/golang/go/issues/28435).

The auto-generated "v1" alias is more a bug in goimports plugin in vscode.

Reply:
Our convention is to use `corev1` as the local alias for `k8s.io/api/core/v1`.  That's the dominant style.